### PR TITLE
[Doc] Fix android documentation

### PR DESCRIPTION
**Pull request by kimjh12**
- fix docs more likely to work for beginners

Signed-off-by: Junhwan Kim <jejudo.kim@samsung.com>

---

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: Documentation/getting-started-android.md [hunk 1]
```diff
@@ -48,13 +48,13 @@ export ANDROID_SDK_ROOT=$ANDROID_SDK
...
 export NNSTREAMER_ROOT=$ANDROID_DEV_ROOT/workspace/nnstreamer
-export ML_API_ROT=$ANDROID_DEV_ROOT/workspace/api
+export ML_API_ROOT=$ANDROID_DEV_ROOT/workspace/api
 ```
 
 ### Install required packages
 Some required packages should be installed as below.
 ```bash
-$ sudo apt install subversion curl pkg-config
+$ sudo apt install subversion curl pkg-config subversion gradle
```
Comment (jaeyun-jung) [inline Documentation/getting-started-android.md:57]: subversion is duplicated

**Review by gichan-jang — APPROVED** [review]
LGTM

File: Documentation/getting-started-android.md (hunk 1 above)
Comment (kimjh12) [reply Documentation/getting-started-android.md]: My eye problem... Thanks!!

**Review by zhoonit — APPROVED** [review]
lgtm!

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**